specs/electra/beacon-chain.md

```diff
@@ -1244,39 +1363,31 @@ def is_valid_deposit_signature(pubkey: BLSPubkey,
     return bls.Verify(pubkey, signing_root, signature)
 ```
 
-###### Modified `add_validator_to_registry`
+###### Modified `process_deposit`
 
-*Note*: The function `add_validator_to_registry` is modified to initialize the validator with a balance of zero and add a pending balance deposit to the queue.
+*Note*: The function `process_deposit` is modified to to use the modified `apply_deposit`.
 
 ```python
-def add_validator_to_registry(state: BeaconState,
-                              pubkey: BLSPubkey,
-                              withdrawal_credentials: Bytes32,
-                              amount: uint64) -> None:
-    index = get_index_for_new_validator(state)
-    validator = get_validator_from_deposit(pubkey, withdrawal_credentials)
-    set_or_append_list(state.validators, index, validator)
-    set_or_append_list(state.balances, index, 0)  # [Modified in Electra:EIP7251]
-    set_or_append_list(state.previous_epoch_participation, index, ParticipationFlags(0b0000_0000))
-    set_or_append_list(state.current_epoch_participation, index, ParticipationFlags(0b0000_0000))
-    set_or_append_list(state.inactivity_scores, index, uint64(0))
-    state.pending_balance_deposits.append(PendingBalanceDeposit(index=index, amount=amount))  # [New in Electra:EIP7251]
-```
-
-###### Modified `get_validator_from_deposit`
+def process_deposit(state: BeaconState, deposit: Deposit) -> None:
```

Comment:
It seems that `process_deposit` is not modified at all. I think we don't have to put it in the Electra spec. I don't think changes on `apply_deposit` make it qualified for this.